opent2t-reference/main.js

```diff
@@ -0,0 +1,283 @@
+var electron = require('electron');
+var app = electron.app;
+var BrowserWindow = electron.BrowserWindow;
+var q = require('q');
+var fs = require('fs');
+require('electron-reload')(__dirname);
+var Opent2tHelper = require("./Opent2tHelper");
+var opent2tHelper = new Opent2tHelper();
```

Comment:
is this standard (lowercase vs uppercase ) on specifying the require and the actual variable for the constructed class?